Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW4/src/com/epam/calculator/test/CtgTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.calculator.test;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CtgTest extends BaseTest {
+	
+	@Test(description = "Simle positive ctg:", dataProvider = "dataForCtgPositive", groups = {"positive"})
+	public void testCtgPositive(double first, double expected) {
+		double ctg = calculator.ctg(first);
```

Comment:
better to name result as result, ctgResult not just ctg

Make sure about all another tests